src/test/java/com/thealgorithms/misc/ThreeSumProblemTest.java

```diff
@@ -0,0 +1,76 @@
+package com.thealgorithms.misc;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ThreeSumProblemTest {
+
+    private ThreeSumProblem tsp;
+
+    @BeforeEach
+    public void setup() {
+        tsp = new ThreeSumProblem(); // Initialize the class before each test
+    }
+
+    @Test
```

Comment:
Change this to "Parameterized Tests" since the tests are the same, only the test data changes.